app/javascript/components/link-organizations.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div>
+    <div
+      class="bg-red-200 w-full p-4 flex items-center"
+      v-if="error"
+    >
+      <img
+        class="p-2"
+        :src="require('../../assets/images/icons/error-icon.svg').default"
+      >
+      <p class="">
+        {{ this.errors }}
+      </p>
+    </div>
+    <div class="flex flex-col items-center min-h-screen mt-10">
+      <div class="flex flex-col items-center justify-center max-w-screen-sm p-10 shadow-2xl">
+        <h1 class="pb-10 text-xl font-bold">
+          Vincula tus organizaciones de Github con Froggo
+        </h1>
+        <div class="flex justify-center">
+          <img
+            class="p-1"
+            :src="require('../../assets/images/froggo-logo-green.svg').default"
+          >
+          <img
+            class="p-1"
+            :src="require('../../assets/images/two-way-arrows.svg').default"
+          >
+          <img
+            class="p-1"
+            :src="require('../../assets/images/github-logo.svg').default"
+          >
+        </div>
+        <p class="px-4 pt-5 pb-8 text-justify">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Labore quis dignissimos,
+          quaerat fuga aspernatur laboriosam ipsum! Magnam, libero, aliquam nihil ducimus repellat,
+          quibusdam inventore.
+        </p>
+        <div
+          v-for="(organization) in organizations"
+          :key="organization.login"
+          class="flex w-5/6"
+        >
+          <froggo-checkbox
+            :id="organization.login"
+            :label="organization.login"
+            :inputValue="organization"
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Attribute ':inputValue' must be hyphenated  vue/attribute-hyphenation

Reply:
Acá para arreglar esto deberías usar `:input-value="organization"